examples/tca_example/.bazelrc

```diff
@@ -6,3 +6,6 @@ import %workspace%/../../ci.bazelrc
 
 # Try to import a local.rc file; typically, written by CI
 try-import %workspace%/../../local.bazelrc
+
+# Set minimum macOS version
+build --macos_minimum_os=10.15 --host_macos_minimum_os=10.15
```

Comment:
Curious. Why is this needed?

Reply:
Fails with this error without this line:

```
dyld[91205]: Symbol not found: _$sSa12_endMutationyyFyXl_Ts5
  Referenced from: <217B6508-EB95-3AF5-9BD0-E76EB0CE1E2B> /private/var/tmp/_bazel_jp/f0f5d2b7b41704886b6a88004e7b18f1/execroot/_main/bazel-out/darwin_arm64-opt-exec-2B5CBBC6-ST-5a29d6cb7794/bin/external/rules_swift_package_manager~override~swift_deps~swiftpkg_swift_case_paths/Sources_CasePathsMacros (built for macOS 14.0 which is newer than running OS)
  Expected in:     <EAA43988-4F04-3D39-A0A6-D597E46E966E> /usr/lib/swift/libswiftCore.dylib
```